lib/src/layout_element.dart

```diff
@@ -112,9 +112,12 @@ class TableLayoutElement extends LayoutElement {
           columni++;
         }
         if (child is TableCellElement) {
+          String width = child.attributes["width"] ?? "";
+          width.replaceAll("px", "");
+          final pxSize = double.tryParse(width) ?? 120;
           cells.add(GridPlacement(
             child: Container(
-              width: double.infinity,
+              width: pxSize,
```

Comment:
This looks like a breaking change. What were you trying to achieve here?

Reply:
Add min width for 'table' tag. 
It's not a good solution, please just force on "Fix issue #834 Inline style with "text-shadow" parse failed."